src/pyedb/dotnet/edb_core/cell/layout.py

```diff
@@ -331,3 +333,21 @@ def find_component_by_name(self, value: str):
         """
         obj = self._pedb._edb.Cell.Hierarchy.Component.FindByName(self._edb_object, value)
         return EDBComponent(self._pedb, obj) if obj is not None else None
+
+    def find_primitive(self, layer_name: Union[str, list]) -> list:
```

Comment:
Union[str, list] is a great feature but I think is not supported in python 3.7 I had to remove last time as it was failing for users. I would check.

Reply:
It's been used in places already. Does pyedbg still support python3.7

Reply:
Yes for few months

Reply:
checked work with python 3.8 closing.